tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,46 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(
+    rd2html("\\link[base]{library}"),
+    link_remote(label = "library", topic = "library", package = "base")
+  )
+})
+
+test_that("can parse local links with topic!=label", {
+  # make a fake index that we can use to search for local help topics as if
+  # we were running build_site on the pkgdown package
+  pkgdownindex = list(
```

Comment:
For testing, it's better to make this sort of throw away data structure as simple as possible.